Automattic/wp-calypso
# Inline components/promo-card into blocks/promo-card-block

#### Proposed Changes

We've run into an issue in the `EmailProvidersStackedComparison` component (accessible at the slug `/email/:domain/purchase/:site_slug`) where the WordPress and Google logos are styled the wrong way *if the user has already visited the Stats pages in the current session*. See attached screencast for clarification.


https://user-images.githubusercontent.com/1101677/183856767-1addf78b-33b8-4201-924e-b25ccbcf1704.mov



This seems to have been due to the fact that there are two different components in the Calypso codebase that use the `.promo-card` CSS class. One in `calypso/components/promo-card/index.jsx` and one in `calypso/components/promo-section/promo-card/index.tsx`. 

Given that `calypso/components/promo-card/index.jsx` is the newest file and that it's only being referenced in a single place (`calypso/blocks/promo-card-block/index.jsx`), I opted to simply inline that `PromoCard` component into `PromoCardBlock` and change the CSS class accordingly. 

#### Testing Instructions



1. Have a website with at least one custom domain that doesn't already have an email subscription
2. Go to the Statistics page
3. Ensure that the Yoast upsell as introduced in #65763 is displayed correctly
4. Go to Upgrades > Email > Add Email (for any domain)
5. Ensure that the page renders correctly (see #64953 for reference)

#### Pre-merge Checklist






Related to #65763
